apps/mobile/app/screens/authenticated/setting-screen/components/timezone-popup.tsx

```
@@ -15,129 +15,122 @@ import {
 export interface Props {
-	visible: boolean
-	onDismiss: () => unknown
-	onTimezoneSelect: (s: string) => unknown
-	userTimezone: string
+	visible: boolean;
+	onDismiss: () => unknown;
+	onTimezoneSelect: (s: string) => unknown;
+	userTimezone: string;
 }
 
 export interface IFilter {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)